src/test/java/uk/gov/justice/digital/delius/service/PersonalCircumstanceServiceTest.java

```diff
@@ -49,16 +51,40 @@ public void personalCircumstancesOrderedNaturallyByPrimaryKeyReversed() {
     public void deletedRecordsIgnored() {
         Mockito.when(personalCircumstanceRepository.findByOffenderId(1L))
                 .thenReturn(ImmutableList.of(
-                        aPersonalCircumstance().toBuilder().personalCircumstanceId(1L).build(),
-                        aPersonalCircumstance().toBuilder().personalCircumstanceId(2L).softDeleted(1L).build(),
-                        aPersonalCircumstance().toBuilder().personalCircumstanceId(3L).build()
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(1L).startDate(LocalDate.now()).build(),
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(2L).startDate(LocalDate.now()).softDeleted(1L).build(),
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(3L).startDate(LocalDate.now()).build()
                 ));
 
         assertThat(personalCircumstanceService.personalCircumstancesFor(1L)
                 .stream().map(uk.gov.justice.digital.delius.data.api.PersonalCircumstance::getPersonalCircumstanceId)
                 .collect(Collectors.toList()))
                 .contains(1L, 3L);
 
+    }
+
+    @Test
+    public void personalCircumstancesHaveActiveFlag() {
+        LocalDate today = LocalDate.now();
+        LocalDate futureDate = LocalDate.now().plusDays(1);
+        LocalDate pastDate = LocalDate.now().minusDays(1);
```

Comment:
Use `final var` when the type is obvious

Reply:
Thanks for this suggestion, I've not encountered much use of var and it looks useful!